<?xml version="1.0"?>
<launch>

  <arg name="processing" default="processing"/>
  <group ns="$(arg processing)">

    <!-- Cloud handler -->
    <node pkg="pmd_camboard_nano" type="pmd_camboard_nano_cloud_normal_estimation" name="cloud_normal_estimation">
          <param name="write_to_file" type="bool" value="true"/>
          <param name="searchRadius" type="double" value="0.3"/>
    </node>
  </group>
</launch>
